src/test/groovy/graphql/DataFetchingErrorGraphQLError.groovy

```diff
@@ -0,0 +1,42 @@
+package graphql
+
+import graphql.language.SourceLocation
+
+class DataFetchingErrorGraphQLError implements GraphQLError {
```

Comment:
not sure, but don't we have this or similar error class

Reply:
I didn't see one and besides, it hardcodes some data that is useful for my tests